City-of-Helsinki/berth-reservations-admin
# VEN-1023 | Improve loading time for single harbor pages

## Description :sparkles:
- Omit firstName & lastName from harbor page query
- Make BerthDetails fetch its data
- The customer name is fetched separately

## Issues :bug:

### Closes :no_good_woman:
[VEN-1023](https://helsinkisolutionoffice.atlassian.net/browse/VEN-1123): Improve loading time for single harbor pages

### Related :handshake:

## Testing :alembic:

### Automated tests :gear:️

### Manual testing :construction_worker_man:
- Go to [Venepaikat page](https://venepaikka-admin.test.kuva.hel.ninja/harbors)
- Search for "Puotilan venesatama"
- Go to the corresponding page, the page should be loaded within 3-10 seconds
- In the piers table, when you expand the row it shows a loading spinner then it fetches the data

## Screenshots :camera_flash:

## Additional notes :spiral_notepad:
- The user is no longer able to sort the piers table by the customer names or search for a customer
